docs/Deployment/Production-Checklist.md

```diff
@@ -10,7 +10,7 @@ deployment.
 
 ## Enable Production Mode
 
-When running Cube.js in production, make sure `NODE_ENV` is set to `production`.
+When running Cube.js Backend in production make sure `NODE_ENV` is set to `production` and `CUBEJS_DEV_MODE` is not set to `true`.
```

Comment:
Can we change the first part of the sentence back to how it was?:

```diff
-When running Cube.js Backend in production
+When running Cube.js in production,
```

Reply:
👍